core/src/main/scala/org/typelevel/sbt/TypelevelPlugin.scala

```diff
@@ -17,23 +40,57 @@ object TypelevelPlugin extends AutoPlugin {
   }
 
   import autoImport._
+  import TypelevelKernelPlugin.autoImport._
   import TypelevelSettingsPlugin.autoImport._
   import GenerativePlugin.autoImport._
   import GitHubActionsPlugin.autoImport._
+  import HeaderPlugin.autoImport._
 
   override def globalSettings = Seq(
     tlFatalWarningsInCi := true
   )
 
   override def buildSettings = Seq(
+    organization := "org.typelevel",
+    organizationName := "Typelevel",
+    startYear := Some(java.time.YearMonth.now().getYear()),
+    licenses += "Apache-2.0" -> url("http://www.apache.org/licenses/"),
     Def.derive(tlFatalWarnings := (tlFatalWarningsInCi.value && githubIsWorkflowBuild.value)),
     githubWorkflowBuildMatrixExclusions ++= {
       for {
         // default scala is last in the list, default java first
-        scala <- (ThisBuild / githubWorkflowScalaVersions).value.init
-        java <- (ThisBuild / githubWorkflowJavaVersions).value.tail
+        scala <- githubWorkflowScalaVersions.value.init
+        java <- githubWorkflowJavaVersions.value.tail
       } yield MatrixExclude(Map("scala" -> scala, "java" -> java.render))
     }
+  ) ++ replaceCommandAlias(
+    "ci",
+    "; project /; headerCheckAll; scalafmtCheckAll; scalafmtSbtCheck; clean; test; mimaReportBinaryIssues"
   )
 
+  override def projectSettings = AutomateHeaderPlugin.projectSettings ++
+    Seq(
+      headerLicense := {
+        // the following was copied from sbt-header
```

Comment:
Why?  What is it doing which that plugin didn't?

Reply:
That's a good question. The comment I didn't add above this was `// copied from sbt-spiewak`.

I suspect you are right, we probably don't need this. @djspiewak has a whole `endYear` thing that I didn't copy here for simplicity (unless we want that?).

Reply:
`endYear` is controversial.  My view is that in most jurisdictions, it only matters when the youngest contributor dies, and we avoid complexity and ritual diffs by omitting it.